macosx/Torrent.mm

```diff
@@ -19,7 +19,9 @@
 #import "NSStringAdditions.h"
 #import "TrackerNode.h"
 
-#define ETA_IDLE_DISPLAY_SEC (2 * 60)
+NSString* const kTorrentDidChangeGroupNotification = @"TorrentDidChangeGroup";
+
+static int const kETAIdleDisplaySec = 2 * 60;
```

Comment:
nit: Prefer an NSTimeInterval here?

Reply:
This variable is later only compared to an `int` so it makes sense to keep the type as it is